File: backend/src/main/java/space/davids_digital/sweetie/rest/exception/InvalidSessionStateException.java
package space.davids_digital.sweetie.rest.exception;

import org.springframework.http.HttpStatus;
import org.springframework.web.bind.annotation.ResponseStatus;

@ResponseStatus(HttpStatus.FORBIDDEN)
public class InvalidSessionStateException extends RuntimeException {
    public InvalidSessionStateException(String message) {
        super(message);
    }

    public InvalidSessionStateException() {
        super();
    }
}

File: backend/src/main/java/space/davids_digital/sweetie/rest/exception/InvalidSessionStateException.kt
package space.davids_digital.sweetie.rest.exception

import org.springframework.http.HttpStatus
import org.springframework.web.bind.annotation.ResponseStatus

@ResponseStatus(HttpStatus.FORBIDDEN)
class InvalidSessionStateException : RuntimeException {
    constructor(message: String?) : super(message)
    constructor() : super()
}
